### PR TITLE
Updates dmitool

### PR DESCRIPTION
**Pull request by alsoandanswer**
title

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Review by Arrow768 — COMMENTED** [review]
Please specify the source of the update.

**Comment by alsoandanswer** [comment]
I just took it from the latest version of Bay

File: tools/dmitool/dmitool.py [hunk 1]
```diff
@@ -1,22 +1,23 @@
-""" Python 3.4 wrapper for dmitool.
+""" Python 2.7 wrapper for dmitool.
```
Comment (Arrow768) [inline tools/dmitool/dmitool.py:1]: Python 2.7 is deprecated.
if that´s the latest thing bay has, you should look at tg.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**